Configurations/README

```diff
@@ -233,7 +233,7 @@ In each table entry, the following keys are significant:
                            apps, as needed by the target.  This code
                            cannot be placed in a library, as the rest
                            of the code isn't expected to link to it
-                           explicitely.
+                           explicitly.
```

Comment:
I see four locations, where this is misspelled. Would you be so kind and fix the remaining, too?

```make
crypto/evp/e_aes.c:2258:         * In case message length was not previously set explicitely via
Configurations/README:236:                           explicitely.
Configurations/common0.tmpl:19:         # Things we explicitely depend on are usually generated
INSTALL:1205: so it is not necessary to explicitely provide this option. Note also
```